lib/googleanalytics.js

```diff
@@ -21,15 +24,70 @@ module.exports = {
     ga('create', process.env.GA_ACCOUNT, 'auto');
     ga('send', 'pageview');
   },
+
   /**
    * pageview:
    * a proxy to basic GA pageview tracking
-   * @param  {[type]} path [description]
-   * @return {[type]}      [description]
+   * @param  {String} path - the current page page e.g. '/about'
    */
   pageview: function (path) {
     if (typeof ga === 'function') {
       ga('send', 'pageview', path);
     }
+  },
+
+  /**
+   * modalview:
+   * a proxy to basic GA pageview tracking
+   * @param  {String} modalName e.g. 'add-or-edit-club'
+   */
+  modalview: function (modalName) {
+    path = '/modal/' + modalName;
```

Comment:
That's interesting, and I've not seen it done before. 

If we make that switch, I think we would have pageview tracking via the router, and we could remove the extra tracking in the modal module.